### DISCUSSION TITLE
Email template for sending welcome email when a user is confirmed

### DISCUSSION THREAD

**Discussion by yunpaik** [supabase/supabase#23287] [Feature Requests]
I want to send a nice welcome/onboarding email only when the user is confirmed, not when they are signed up. It would be very helpful if we can have an email template that gets sent out when the user is confirmed.

Currently, as a workaround, I am assigning a special flag to user_metadata when a user is signed in for the first time. Then there's a webhook listening to the UPDATE event on the auth.users table. When the value of the special flag is set to true from false, I am sending out a welcome email. This works but is extremely inefficient because this event will be triggered every time a user logs in, since `last_sign_in_at` field gets updated.

I also looked into listening to a change in `confirmed_at` field in the users table, but I never get an UPDATE event when `confirmed_at` is updated if a user logs in with Google OAuth.

**Comment by charliemday** [comment]
Could it be because `confirmed_at` is a computed field and what you really need is `email_confirmed_at`?
